packages/dd-trace/src/plugins/tracing.js

```diff
@@ -66,31 +66,35 @@ class TracingPlugin extends Plugin {
     this.addError(ctxOrError)
   }
 
-  addTraceSubs () {
+  registerOperation (channelName) {
```

Comment:
Why the alias? The way this is named I would expect it takes only an operation name and do some additional magic. Otherwise it's redundant.